cmd/acr/purge.go

```diff
@@ -502,11 +534,7 @@ func dryRunPurge(ctx context.Context, acrClient api.AcrCLIClientInterface, login
 			for _, tag := range *tagsToDelete {
 				// For every tag that would be deleted first check if it exists in the map, if it doesn't add a new key
 				// with value 1 and if it does just add 1 to the existent value.
-				if _, exists := deletedTags[*tag.Digest]; exists {
-					deletedTags[*tag.Digest]++
-				} else {
-					deletedTags[*tag.Digest] = 1
-				}
+				deletedTags[*tag.Digest]++
```

Comment:
Q: what is the reason of removing the exists check? Also if you remove the check what do you think of also removing the above comment?

Reply:
golinter was complaining about the previous check, so I changed it to what it recommended.

Reply:
The comment is still relevant, if the entry doesn't exist then it will add one and assign it 1